docs/guides/react-server-cli.md

```diff
@@ -157,7 +157,8 @@ export default (server, reactServerMiddleware) => {
 	server.use(compression());
 	server.use(bodyParser.urlencoded({ extended: false }));
 	server.use(bodyParser.json());
-	server.use(session(options));
+	server.use(helmet());
```

Comment:
Oh, I see, and users are responsible for adding it themselves if they override the default with custom middleware.

Nice catch.